src/MICore/project.json

```diff
@@ -32,7 +34,8 @@
     "System.Threading.Thread": "4.0.0",
     "System.Threading.Timer": "4.0.1",
     "System.Xml.ReaderWriter": "4.0.11",
-    "System.Xml.XmlSerializer": "4.0.11"
+    "System.Xml.XmlSerializer": "4.0.11",
+    "Newtonsoft.Json": "9.0.1"
```

Comment:
>Newtonsoft.Json [](start = 5, length = 15)

The version of Newtonsoft.Json.dll in devenv has an assembly version of 8.0.0.0 for VS 2017 and 6.0.0.0 for VS 2015. So you need to either compile again an older version of this, or update Install.cmd + targets now, and SetupPackages when you checkin to ship yet-another-copy of this dll.